docs/content/doc/development/oauth2-provider.zh-cn.md

```diff
@@ -0,0 +1,96 @@
+---
+date: "2019-04-19:44:00+01:00"
+title: "OAuth2 提供者"
+slug: "oauth2-provider"
+weight: 41
+toc: false
+draft: false
+menu:
+  sidebar:
+    parent: "development"
+    name: "OAuth2 提供者"
+    weight: 41
+    identifier: "oauth2-provider"
+---
+
+# OAuth2 提供者
+
+**目录**
+
+{{< toc >}}
+
+Gitea 支持作为 OAuth2 提供者，能够让第三方应用在用户同意的情况下读写 Gitea 的资源。此功能自 1.8.0 版本开始支持。
+
+## Endpoint
+
+| Endpoint               | URL                         |
+| ---------------------- | --------------------------- |
+| Authorization Endpoint | `/login/oauth/authorize`    |
+| Access Token Endpoint  | `/login/oauth/access_token` |
+
+## 支持的 OAuth2 Grant
+
+目前 Gitea 只支持 [**Authorization Code Grant**](https://tools.ietf.org/html/rfc6749#section-1.3.1) 标准，并额外支持下列扩充标准：
+
+- [Proof Key for Code Exchange (PKCE)](https://tools.ietf.org/html/rfc7636)
+- [OpenID Connect (OIDC)](https://openid.net/specs/openid-connect-core-1_0.html#CodeFlowAuth)
+
+若想要让第三方应用使用 Authorization Code Grant，需要先在「设置」(`/user/settings/applications`)中注册一个新的应用程序。
+
+## Scope
+
+目前 Gitea 尚未支持 scope （参见 [#4300](https://github.com/go-gitea/gitea/issues/4300)），所有的第三方应用都可获得该用户以及他所属的组织中所有资源的读写权。
```

Comment:
#4300 has been resolved in 1.19.0

Reply:
A new commit was submitted, I translate it again from the latest en-us documents.